### ISSUE TITLE
`Error: Invalid response id: undefined` when using streamOperations

### ISSUE THREAD

**Issue by MrToph** [steemit/steem-js#322]
Hi,

I'm using `steem 0.67` and the streamOperations

```js
steem.api.streamOperations(`irreversible`, (err, res) => {
      if (err) {
        console.log(err);
      }
    })
```

And I get an `Error: Invalid response id: undefined` error, since yesterday.
Before it worked fine.

I'm using ``steem.api.setOptions({ url: `https://api.steemit.com` })``

Is anyone experiencing the same?

**Comment by Slapbox** [comment]
Yep same thing here. Started roughly the same date. No issues before that date.

**Comment by ety001** [comment]
This repository has been superseded by a complete 2025 rewrite. The new trunk (`master`, starting at v1.1.0) is a ground-up rewrite of `@steemit/steem-js` and the legacy 0.7.x / 0.8.x code line is archived.

As part of that transition, this issue is being closed because:
- It was filed against the legacy codebase, and the code it references no longer exists in the rewritten trunk.
- The new codebase reimplements the affected areas (RPC routing, transaction serialization, auth/signing, crypto, browser builds) from scratch, so this issue cannot be carried over as-is.

If the problem or request still applies to the rewritten codebase (v1.1.0+ on `master`), please open a new issue against the current trunk with details that reproduce on the new version. Thank you for taking the time to report this originally.
